src/ocaml-output/Makefile

```diff
@@ -89,7 +96,6 @@ endif
 	                    -e '/%type.*->.*/d' \
 	                    >> $@
 
-.PHONY: fstar.exe
```

Comment:
I'd rather keep that line on (weird ocamlbuild/windows/mtimes issue, if I recall correctly)